docs/setup/upgrade/index.md

```diff
@@ -55,6 +55,20 @@ $ helm upgrade stash appscode/stash \
     --set-file global.license=/path/to/the/license.txt
 ```
 
+#### 3. Post uprade cleanup
+
+If you were using Stash Auto-backup with cross-namespace-repository, you may have existing Repositories in the `repoNamespace` and BackupNamespace in your application's namespace. If you use `backupNamespace` in Auto-Backup, those previously existing Repositories will create conflict. In this case, you need to clean up the old BackupConfigurations and Repositories manually.
+
```

Comment:
We have changed the name format for auto-backup resources to support dedicated backup or storage namespace. If you were using dedicated storage namespace for your auto-backup through `repoNamespace` field, you might see new BackupConfiguration and Repository has been created with new name format. Unfortunately, it is not possible for Stash to remove the old BackupConfiguration and Repository before creating new one due a flaw in how it them previously. In this case, you can safely remove the old BackupConfiguration and Repository. Your backed up data will be intact.